doc/source/ray-core/handling-dependencies.rst

```diff
@@ -506,11 +506,11 @@ Your ``runtime_env`` dictionary should contain:
 
 .. warning::
 
-  Check for hidden files and metadata directories (e.g. ``__MACOSX/``) in zipped dependencies.
+  Check for hidden files and metadata directories in zipped dependencies.
   You can inspect a zip file's contents by running the ``zipinfo -1 zip_file_name.zip`` command in the Terminal.
   Some zipping methods can cause hidden files or metadata directories to appear in the zip file at the top level.
-  This will cause Ray to throw an error because the structure of the zip file is invalid since there is more than a single directory at the top level.
   You can avoid this by using the ``zip -r`` command directly on the directory you want to compress. Make sure to run the command from that directory's parent.
+  If Ray detects more than a single directory at the top level, it will use the entire zip file instead of the top-level directory, which may lead to unexpected behavior.
```

Comment:
To avoid this, use the ``zip -r`` command directly on the directory you want to compress from its parent's directory. For example, if you have a directory structure such as: `a/b` and you what to compress `b`, issue the ``zip -r b`` from the directory `a.`

Reply:
I assume that's what you mean with zip -r?

Reply:
Yes, that's correct and the way you've stated it is cleaner.  I'll update this.